common/formatting/layout_optimizer_test.cc

```diff
@@ -667,9 +667,25 @@ class LayoutFunctionFactoryTest : public ::testing::Test,
             "Exactly at 30 limit (30 cols).\n"
             "One over 30 limit (31 columns).\n"
 
+            "10 columns\r"
```

Comment:
mmh, instead of repeating the same thing (which is harder to manually verify or extend), can we formulate that in a way that it comes from a single source (take the \n-separated version and create the \r version from it). Also that will allow us to determine `kWrappable` programmatically.